packages/serverless/test/util/AbstractServerlessControl.js

```diff
@@ -13,7 +13,7 @@ const path = require('path');
 const fetch = require('node-fetch-v2');
 
 const retry = require('@instana/core/test/test_util/retry');
-const delay = require('@instana/core/test/test_util/delay');
+// const delay = require('@instana/core/test/test_util/delay');
```

Comment:
Instead of commenting, please remove this as we no longer use the delay.

Reply:
Yes will do. I was waiting for the tests to pass on CI :)

Reply:
There was a weird timeout for the backend stub for 2s, no idea why.